toolkit/tools/scheduler/schedulerutils/buildworker.go

```diff
@@ -99,6 +101,7 @@ func BuildNodeWorker(channels *BuildChannels, agent buildagents.BuildAgent, grap
 		res := &BuildResult{
 			Node:           req.Node,
 			AncillaryNodes: req.AncillaryNodes,
+			WasDelta:       req.IsDelta,
```

Comment:
I'm curious, do we need the new `IsDelta` and `WasDelta` fields, if we're passing around the node itself and it already contains the information if it's a delta node or not inside `node.State`.